test/connectivity/modules/NetworkController/docker/linux/arm64v8/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG base_tag=1.0.5.1-linux-arm64v8
-FROM edgebuilds.azurecr.io/microsoft/azureiotedge-module-base-full:${base_tag}
+FROM azureiotedge/azureiotedge-module-base-full:${base_tag}
```

Comment:
Would you remind me what are the versions of this module-base-full used in master, 1.0.10 and 1.0.9?

Reply:
master = module-base-full 1.0.6.0
release1.0.10 = module-base-full 1.0.5.1 (this PR)
release1.0.9 = module-base-rocksdb 1.0.4.1